crosswalk/index.html

```diff
@@ -147,9 +147,9 @@ <h2>EPUB and ONIX</h2>
 					</tr>
 					<tr>
 						<td>
-							<p><a href="https://www.w3.org/TR/epub-a11y/#sec-conf-reporting-pub"
-										><code>dcterms:conformsTo</code></a> with the URL
-									<code>http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a</code></p>
+                            <p><a href="https://www.w3.org/TR/epub-a11y/#sec-conf-reporting-pub"
+										><code>dcterms:conformsTo</code></a> with the text string<br/>
+                                <code>EPUB Accessibility 1.0 - WCAG 2.0 Level A</code></p>
```

Comment:
These 1.0 text strings aren't valid per the 1.0 or 1.1 specifications, so if people were to use these they wouldn't conform to either. They have to map to the old IDPF URLs for 1.0 conformance.